probe-rs/targets/STM32C0_Series.yaml

```diff
@@ -14,7 +16,7 @@ variants:
       psel: 0x0
   memory_map:
   - !Nvm
-    name: BANK_1
+    name: Flash
```

Comment:
STM memory maps should be generated from embassy data using https://github.com/bugadani/stm-probers

Reply:
We should put that into the instructions somewhere. Sorry I forgot about that.

Reply:
This is a typical "Here be dragons" task to be fair